apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/pac/ClaimE2E.java

```diff
@@ -37,6 +46,147 @@ public static void tearDown() {
+   * Tests to see if the correct response is given when a search is done for {@link Claim}s using
+   * given mbi and service-date range. In this test case the query finds the matched claims because
+   * their to dates are within the date range even though their from dates are not.
+   */
+  @Test
+  public void shouldGetCorrectClaimResourcesByMbiHash() {
+    String requestString =
+        claimEndpoint
+            + "?mbi="
+            + RDATestUtils.MBI_HASH
+            + "&service-date=gt1970-07-18&service-date=lt1970-07-25";
+
+    String response =
+        given()
+            .spec(requestAuth)
+            .expect()
+            .statusCode(200)
+            .when()
+            .get(requestString)
+            .then()
+            .extract()
+            .response()
+            .asString();
+
+    String expected = rdaTestUtils.expectedResponseFor("claimSearch");
+
+    Set<String> ignorePatterns = new HashSet<>(IGNORE_PATTERNS);
+    ignorePatterns.add("\"/id\"");
+    ignorePatterns.add("\"/entry/[0-9]+/resource/created\"");
+
+    AssertUtils.assertJsonEquals(expected, response, ignorePatterns);
+  }
+
+  /**
+   * Tests to see if the correct response is given when a search is done for {@link Claim}s using
+   * given mbi and service-date range with tax numbers included. In this test case the query finds
+   * the matched claims because their to dates are within the date range even though their from
+   * dates are not.
+   */
+  @Test
+  public void shouldGetCorrectClaimResourcesByMbiHashWithTaxNumbers() {
+    String requestString =
+        claimEndpoint
+            + "?mbi="
+            + RDATestUtils.MBI_HASH
+            + "&service-date=gt1970-07-18&service-date=lt1970-07-25";
+
+    String response =
+        given()
+            .spec(requestAuth)
+            .header(CommonHeaders.HEADER_NAME_INCLUDE_TAX_NUMBERS, "true")
+            .expect()
+            .statusCode(200)
+            .when()
+            .get(requestString)
+            .then()
+            .extract()
+            .response()
+            .asString();
+
+    String expected = rdaTestUtils.expectedResponseFor("claimSearchWithTaxNumbers");
+
+    Set<String> ignorePatterns = new HashSet<>(IGNORE_PATTERNS);
+    ignorePatterns.add("\"/id\"");
+    ignorePatterns.add("\"/entry/[0-9]+/resource/created\"");
+
+    AssertUtils.assertJsonEquals(expected, response, ignorePatterns);
+  }
+
+  /**
+   * Tests to see if the correct paginated response is given when a search is done for {@link
+   * Claim}s using given mbi and service-date range. In this test case the query finds the matched
+   * claims because their from dates are within the date range even though their to dates are not.
+   */
+  @Test
+  public void shouldGetCorrectClaimResourcesByMbiHashWithPagination() {
+    String requestString =
+        claimEndpoint
+            + "?mbi="
+            + RDATestUtils.MBI_HASH
+            + "&service-date=gt1970-07-18&service-date=lt1970-07-25"
```

Comment:
just checking in that the date range change from `List.of(new DateParam("ge1970-07-10"), new DateParam("le1970-07-18"))`  isn't significant here :)

Reply:
Good eye, I didnt notice that one was different. The test passes with the changed date, so I guess it's not, but I'll change it back just to reduce the changes between the tests.